src/lang/es.json

```diff
@@ -7,226 +7,226 @@
   "% Threshold heart rate": "% Umbral de la frecuencia cardíaca",
   "% eFTP": "% eFTP",
   "%FTP": "%FTP",
-  "%FTP Range": "%FTP Rango",
+  "%FTP Range": "Rango %FTP",
   "%LTHR": "%LTHR",
-  "%LTHR Range": "%LTHR Rango",
+  "%LTHR Range": "Rango %LTHR",
   "%MMP": "%MMP",
-  "%MMP Range": "%MMP Rango",
+  "%MMP Range": "Rango %MMP",
   "%MaxHR": "%MaxHR",
-  "%MaxHR Range": "%MaxHR Rango",
-  "%Pace": "%Pace",
+  "%MaxHR Range": "Rango %MaxHR",
+  "%Pace": "%Ritmo",
   "%Pace Range": "Rango de ritmo",
   "1 week": "1 semana",
-  "1 year": "1 AÑO",
-  "100% flat out sprint for the line": "100% a toda velocidad para la línea",
+  "1 year": "1 año",
+  "100% flat out sprint for the line": "100% a toda velocidad para la línea *",
   "10s Power": "10s Potencia",
   "180 days": "180 días",
   "30s Power": "30s Potencia",
   "30s power exponent": "Exponente de potencia de 30s",
-  "3s Power": "Potencia de 3s",
-  "42 day moving average": "Media móvil de 42 días",
-  "42 day moving total": "Total móvil de 42 días",
+  "3s Power": "3s Potencia",
+  "42 day moving average": "42 días tiempo en movimiento promedio",
```

Comment:
I have chosen the 'promedio' word in lieu of 'media', because media can have another meanings when talking about statistics. 'Promedio' it always and only means 'Average'